tests/odb/loose.c

```diff
@@ -103,6 +141,22 @@ void test_odb_loose__simple_reads(void)
 	test_read_object(&some);
 }
 
+void test_odb_loose__streaming_reads(void)
+{
+	size_t blocksizes[] = { 1, 2, 4, 16, 99, 1024, 123456789, 0 };
```

Comment:
Ugh, I was really confused by the `0` blocksize here. Didn't see that it was used as a terminator, so I kept on asking myself for several minutes how we can actually fully read through the stream in that case. Maybe remove that blocksize and iterate by array size instead of by sentinel?

Reply:
Yeah, I think you're right; I fixed this.